openreview/venue_request/venue_request.py

```diff
@@ -998,21 +998,27 @@ def setup_request_form(self):
                 'default': 'No, wait until the submission deadline has passed to make them public.',
                 'order': 24
             },
+            'withdraw_submission_expiration': {
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
+                'description': 'By when authors can withdraw their submission? Please specify the due date in GMT using the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59)',
```

Comment:
```suggestion
                'description': 'By when authors can withdraw their submission? Please specify the expiration date in GMT using the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59)',
```

Reply:
I might reword this to "When should authors lose the option to withdraw their submissions?"